### PR TITLE
Patch #90 (pep8 issues, docstring consistency)

### PR DESCRIPTION
**Pull request by yardasol**
## Summary of changes

#90 had some pep8 issues and docstring inconsistencies that were not addressed before merging. This PR addresses those issues.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Required for Merging
- [x] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
  - [ ] CI tests pass
  - [ ] Local tests pass (including Serpent2 integration tests)

## Associated Issues and PRs

- PR: #90 

## Associated Developers

- Dev: @

## Checklist for Reviewers

Reviewers should use [this link](https://arfc.github.io/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

File: saltproc/depcode.py [hunk 1]
```diff
@@ -175,6 +165,29 @@ def __init__(self,
...
+           ----------
+           exec_path : str
+               Path to Serpent2 executable.
+           template_path : str
+               Path to user input file for Serpent2.
+           input_path : str
+               Name of input file for Serpent2 rerunning.
+           iter_matfile : str
+               Name of iterative, rewritable material file for Serpent2
+               rerunning. This file is modified during  the simulation.
```
Comment (LukeSeifert) [inline saltproc/depcode.py:179]: Consider adding the optional tag to these parameters

File: saltproc/depcode.py [hunk 2]
```diff
@@ -314,12 +327,12 @@ def get_tra_or_dec(self, input_file):
 
         Parameters
         -----------
-        input_file: str
+        input_file : str
             Serpent2 input file name and path.
 
         Returns
         --------
```
Comment (LukeSeifert) [inline saltproc/depcode.py:334]: There is one extra "-" on the `Parameters` and the `Returns` sections

File: saltproc/depcode.py [hunk 3]
```diff
@@ -186,25 +199,25 @@ def __init__(self,
                          active_cycles,
                          inactive_cycles)
 
-    def change_sim_par(self, data):
+    def change_sim_par(self, template_data):
```
Comment (LukeSeifert) [inline saltproc/depcode.py:202]: `template_data` is a much better variable name, nice work

**Review by LukeSeifert — COMMENTED** [review]
I made a few minor comments, but overall I think it looks good.

File: saltproc/depcode.py (hunk 2 above)
Comment (yardasol) [reply saltproc/depcode.py:334]: Nice catch

File: saltproc/depcode.py (hunk 1 above)
Comment (LukeSeifert) [reply saltproc/depcode.py:179]: I was thinking the default values would typically work, but yeah I think you're right that these aren't really optional.

**Review by LukeSeifert — APPROVED** [review]
Looks good!

**Pull request merged**